pkgs/development/python-modules/langdetect/default.nix

```diff
@@ -0,0 +1,18 @@
+{ buildPythonPackage, fetchzip, six, lib }:
+
+buildPythonPackage {
+  pname = "langdetect";
+  version = "1.0.7";
+  src = fetchzip {
```

Comment:
`fetchPyPi`
Maybe you need to pass in a different `pname` here?

Reply:
Maybe it's because there's only a [zip](https://pypi.org/project/langdetect/#files)?

Reply:
extension = "zip";

Reply:
worked